Frameworks/include/IwMalloc.h

```diff
@@ -18,6 +18,8 @@
 #define __IWMALLOC_H
 
 #include <string.h>
+
+#ifdef _WIN32
 #include <WinObjCRTExport.h>
```

Comment:
Can WinObjCRTExport be shored up to be safe here? We might just want to define `WINOBJCRT_IMPEXP=` when building, and then scope down this ifdef.

We could use strncpy for strcpy_s as well :)